README.md

```diff
@@ -28,15 +26,20 @@ Before running tests, ensure a valid Postgres database is running. The simplest
 docker-compose up
 ```
 
-This will start a postgress database. Running test or coverage runs will automatically populate the
+This will start a Postgres database. Running test or coverage runs will automatically populate the
 database with the information it needs.
 
+-   **Note:** you can also run `docker-compose up -d` to run it in the background. You'll can then tail the Postgres logs if needed with `docker-compose logs --tail=100 -f`
+
```

Comment:
what about changing it this way, suggesting by default to run it in the background, and provide a way to get the logs :

<hr>

## Testing, benching & linting
Before running tests, ensure a valid Postgres database is running. The simplest way to do this is via Docker. Assuming docker is installed on your machine, in the root folder, run:

```
docker-compose up -d
```

This will start a Postgres database. Running test or coverage runs will automatically populate the
database with the information it needs.

-   **Note:** you can tail the Postgres logs if needed with `docker-compose logs --tail=100 -f`